src/components/Loading Screen/LoadingScreen.stories.js

```diff
@@ -0,0 +1,12 @@
+
+import LoadingScreen from './LoadingScreen'
+export default {
+    title: 'LoadingScreen',
+    component: LoadingScreen
+}
+  
+const Template = args => <LoadingScreen {...args} />
+  
+export const Default = Template.bind({})
+Default.args = {}
```

Comment:
More examples to see how it looks like in different scenarios would be awesome.

Reply:
Thank you for the comment, but I will add more scenarios as i need them.